src/main/kotlin/net/lumalyte/lg/infrastructure/persistence/invitations/GuildInvitationRepositorySQLite.kt

```
@@ -0,0 +1,196 @@
+
+            results.mapNotNull { row ->
+                try {
+                    GuildInvitation(
+                        guildId = UUID.fromString(row.getString("guild_id")),
+                        guildName = row.getString("guild_name"),
+                        invitedPlayerId = UUID.fromString(row.getString("invited_player_id")),
+                        inviterPlayerId = UUID.fromString(row.getString("inviter_player_id")),
+                        inviterName = row.getString("inviter_name"),
+                        timestamp = Instant.parse(row.getString("invited_at"))
+                    )
+                } catch (e: Exception) {
```

detekt (1.23.8)

[warning] The caught exception is too generic. Prefer catching specific exceptions to the case that is currently handled.

(detekt.exceptions.TooGenericExceptionCaught)